app/models/concerns/smooch_status.rb

```diff
@@ -32,14 +28,6 @@ def get_status_message_for_language(status, language)
           Rails.cache.write("SmoochUserSlackChannelUrl:Team:#{a.team_id}:#{value['id']}", self.value) unless value.blank?
         end
       end
-
-      protected
-
-      def send_message
-        pm = self.annotation.annotated
-        return unless Bot::Smooch.team_has_smooch_bot_installed(pm)
-        ::Bot::Smooch.delay_for(1.second, { queue: 'smooch_priority', retry: 0 }).send_message_on_status_change(pm.id, self.value, self.class.actor_session_id)
-      end
```

Comment:
Why you removed this method, @melsawy ? This is not the one related to the Slack message... this is the one that sends a message to the tipline user when the status changes. We need to keep it.

Reply:
Back this method